### ISSUE TITLE
Node Selector Considerations

### ISSUE THREAD

**Issue by TehTurk** [Frooxius/NeosPublic#1067]
This is for the future so you can categorize this but issues originally themselves were closed, so listing them here in context.

#1065 and #1062, if other considerations are required, please list them here.

**Comment by mralext20** [comment]
re: 1062: if there are colors for types in one place, they should be in most places.

**Comment by Frooxius** [comment]
Yes, I'd definitely like to add general color/pattern coding across the board, as well as icons to help distinguish common components.

**Comment by mralext20** [comment]
i saw @3x1t_5tyl3 build a mock for a node panel, here's what it looked like: 
https://this-is-a.stale-me.me/i/vb8p.png
https://this-is-a.stale-me.me/i/uewp.png

what i like about this implimentation is that there is a listing of folders / categories on the left side

**Comment by Frooxius** [comment]
Having a folder-tree view is planned for the proper one. I plan on building a generalized folder-tree UI element, which will be used for Inventory, File Browser, Component browser and  any other things that could use it and could be Facetized.
